src/gui_common/QuickLoadHandler.tscn

```diff
@@ -0,0 +1,20 @@
+[gd_scene load_steps=2 format=2]
+
+[ext_resource path="res://src/gui_common/QuickLoadHandler.cs" type="Script" id=1]
+
+[node name="QuickLoad" type="Control"]
+visible = false
```

Comment:
Shouldn't this be visible as the child is the one that is hidden? If that causes problems then put the mouse ignore flags on the Control.